api/db/repositories/verification/verification.js

```diff
@@ -0,0 +1,24 @@
+const Customers = require('@db/dbHelper').Customers;
+const { IdNotFound } = require('@helpers/errors');
+
+class Verification {
+    static async DataChecker (checkedInfo, requestedId) {
+        switch (checkedInfo) {
+            case 'customer':
+                const requestedCustomer = await Customers.findById(requestedId);
```

Comment:
this should be happening within IdExists method so that you will pass only actual ID to it and it will do all the job